docs/reference/configuration/configuration.md

```diff
@@ -7,7 +7,20 @@ The Arm Mbed OS configuration system, a part of the Arm Mbed OS build tools, cus
 - The receive buffer size of a serial communication library.
 - The flash and RAM memory size of an Mbed target.
 
-The Arm Mbed OS configuration system gathers and interprets the configuration defined in the target in its [target configuration](../reference/adding-and-configuring-targets.html), all `mbed_lib.json` files and the `mbed_app.json` file. The configuration system creates a single header file, `mbed_config.h`, that contains all of the defined configuration parameters converted into C preprocessor macros. `mbed compile` places `mbed_config.h` in the build directory, and `mbed export` places it in the application root. `mbed compile` runs the Mbed configuration system before invoking the compiler, and `mbed export` runs the configuration system before creating project files.
+Here is how a default JSON file might look like:
+
```

Comment:
I like the example. The text below sort of goes into what this will produce but might also be nice to have a small statement before it saying what the actual end effect will be (...default JSON file that configures X for all targets and Y for the K64F might look like:)

Reply:
I like it. Let me modify accordingly.